src/modules/Packages.rb

```diff
@@ -29,6 +29,13 @@ class PackagesClass < Module
     # Minimum set of packages required for installation with remote X11 server
     REMOTE_X11_BASE_TAGS = [ "xorg-x11-server", "xorg-x11-fonts", "icewm" ]
 
+    # Some products are already be "included" in other products. So they MUST
+    # not be installed anymore because they are conflicting to the other one.
+    PRODUCT_CONFLICTS = {
+      # SLES_SAP includes SLES. So SLES is not needed anymore
```

Comment:
I'd add more details about the conflicts here, e.g.:

```ruby
# SLES_SAP contains "Conflicts: sles-release"
# see https://build.suse.de/package/view_file/SUSE:SLE-12-SP2:GA/_product/SLES_SAP-release.spec?expand=1
```

So everybody can see how the conflict is defined and it is possible to check it in the future whether the conflict is still there.